introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello, my name is richardkoszegi and I like CodeSchool. Every course, that I took, is very good.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?